in_toto/envelope_test.go

```diff
@@ -176,6 +176,16 @@ func TestEnvelopeVerifySignature(t *testing.T) {
 		err = env.VerifySignature(key)
 		assert.NotNil(t, err)
 	})
+
+	t.Run("invalid key", func(t *testing.T) {
+		key := Key{
+			KeyID:   "invalid",
+			KeyType: "invalid",
+		}
+
+		err := env.VerifySignature(key)
+		assert.NotNil(t, err)
```

Comment:
nit: hmm...just a suggestion would be to check the error you are expecting to come back and not something else.